tests/integration/ha_tests/helpers.py

```diff
@@ -730,3 +732,158 @@ async def update_pebble_plans(ops_test: OpsTest, override: Dict[str, str]) -> No
         )
         ret_code, _, _ = await ops_test.juju(*replan_cmd)
         assert ret_code == 0, f"Failed to replan for unit {unit.name}"
+
+
+async def are_all_db_processes_down(ops_test: OpsTest, process: str) -> bool:
+    """Verifies that all units of the charm do not have the DB process running."""
+    app = await get_application_name(ops_test, APP_NAME)
+
+    if "/" in process:
```

Comment:
is it possible to add a comment here on what exactly this check does or maybe create a with an explanatory name ?

Reply:
sure thing!